### PR TITLE
Round 2 Bugfixes

### LINKED ISSUE #12886: Th'akhist loadout item issues.
I have found issues with three items

-Shaman staff: Spawns inside bag despite not being able to fit in there naturally, and takes up an enormous amount of space while inside (almost half of the default bag capacity) 
-Th'akhist head wrappings: No on-character sprite
-Th'akhist wristguards: Spawns worn by your character, but is unable to be re-equipped if you take them off.  Possibly related to improperly set species restrictions.

### LINKED ISSUE #13987: Zavodskoi black cap is missing its object sprite.
This one in the loadout:
image
It'll show up on your head, or inhand when holding it, but it's invisible.

### LINKED ISSUE #14205: Outdated tip(s) of the round
image
There are likely other examples, someone just needs to go over em.

### PR DESCRIPTION
**Pull request by KingOfThePing**
Fixes #13987 
Fixes #12886
Fixes #14205 

If I missed anything in the Tips of the Round, let me know. I mostly scrubbed old references, some stupid wording and didn't anything new.

### CODE REVIEW

**Comment by KingOfThePing** [comment]
!bugfix

**Comment by KingOfThePing** [comment]
!review

File: config/example/tips.txt [hunk 1]
```diff
@@ -12,78 +12,77 @@ You can grab someone by clicking on them with the grab intent, then upgrade the
...
 As the Bartender, you can use a circular saw on your shotgun to make it easier to store.
 As a Janitor, if someone steals your janicart, you can instead use your space cleaner spray, grenades, water sprayer or order another from Cargo.
 As a Janitor, mousetraps can be used to create bombs or booby-trap containers.
 As the Librarian, be sure to keep the shelves stocked and the library clean for the crew.
-As a Cargo Technician, you can hack MULEbots to make them faster, run over people in their way, and even let you ride them!
-As a Cargo Technician, you can earn more cargo points by shipping back crates from maintenance, liquid containers, plasma sheets, rare seeds from hydroponics, and more!
-As the Quartermaster, be sure to check the manifests on crates you receive to make sure all the info is correct. If there's a mistake, stamp the manifest DENIED and send it back in a crate with the items untouched for a refund!
+As a Hangar Technician, you can hack MULEbots to make them faster, run over people in their way, and even let you ride them!
```
Comment (Doxxmedearly) [inline config/example/tips.txt:68]: We don't even like have MULEbots anymore, even on Aurora we didn't, I'm pretty sure. Can just delete this.

File: config/example/tips.txt [hunk 2]
```diff
@@ -93,24 +92,24 @@ As a Cultist, the Blood Boil rune will deal massive amounts of brute damage to n
...
-Sometimes you won't be able to avoid dying no matter how good you are at the game. Try not to stress too much about it.
+To defeat your enemy, shoot at them until they die and go horizontal.
+Sometimes you won't be able to avoid dying no matter how good you are at the game. it's part of the game, so try not to stress too much about it.
 When a round ends nearly everything about it is lost forever, leave your salt behind with it.
-Killing the entire station isn't fun except when it is.
+Killing the entire ship isn't fun except when it is.
 You can win a pulse rifle from the arcade machine. Honest.
 Just like real life the entropy of the game can only increase with time. If things aren't on fire yet, just wait.
-Completing your objectives is good practice, but the best antagonists will strive to do more than the bare minimum to really leave an impression.
+Completingthe objectives you set yourself is good practice, but the best antagonists will strive to do more than the bare minimum to really leave an impression.
```
Comment (Doxxmedearly) [inline config/example/tips.txt:101]:
```suggestion
Setting your own objectives and working to complete them is good practice, but the best antagonists will strive to do more than the bare minimum to really leave an impression.
```

File: config/example/tips.txt [hunk 3]
```diff
@@ -93,24 +92,24 @@ As a Cultist, the Blood Boil rune will deal massive amounts of brute damage to n
...
 Sometimes a round will just be a bust. C'est la vie.
 This is a game that is constantly being developed for. Expect things to be added, removed, fixed, and broken on a daily basis.
 It's fun to try and predict the round type from the tip of the round message.
-The quartermaster is not a head of staff and will never be one.
+The Quartermaster has finally become a Head of Staff. He only needed to be renamed. It can be that easy, kids.
 Your sprite represents your hitbox, so that floor length braid makes you easier to kill. The sacrifices we make for snowflake points.
 Spacemen can't move diagonally but most animals can. Blame the slow decline of the numpad.
 Sometimes admins will just do stuff. Roll with it.
-The remake will never come out.
+The remake will maybe even come out.
```
Comment (Doxxmedearly) [inline config/example/tips.txt:112]:
```suggestion
The remake might come out.
```

File: config/example/tips.txt [hunk 4]
```diff
@@ -93,24 +92,24 @@ As a Cultist, the Blood Boil rune will deal massive amounts of brute damage to n
 As a Cultist, you can create an army of manifested goons using a combination of the Manifest rune, which creates homunculi from ghosts, and the Blood Drain rune, which drains life from anyone standing on any blood drain rune.
 As a Ghost, you can double click on people, bots, or the singularity to follow them.
 Occasionally the tip of the round might lie to you. Don't panic, this is normal.
-To defeat your enemy, shoot at them until they die.
-Sometimes you won't be able to avoid dying no matter how good you are at the game. Try not to stress too much about it.
+To defeat your enemy, shoot at them until they die and go horizontal.
```
Comment (Doxxmedearly) [inline config/example/tips.txt:95]:
```suggestion
To defeat your enemy, click on them until they go horizontal.
```

File: config/example/tips.txt [hunk 5]
```diff
@@ -12,78 +12,77 @@ You can grab someone by clicking on them with the grab intent, then upgrade the
...
-As a Security Officer, your sechuds or HUDsunglasses can not only see crewmates' job assignments and criminal status, but also if they are loyalty implanted. Use this to your advantage in a revolution to definitively tell who is on your side!
-As a Security Officer, examining someone while wearing sechuds or HUDsunglasses will let you set their arrest level, which will cause Beepsky and other security bots to chase after them.
+As a Security Officer, your sechuds or HUDsunglasses can not only see crewmates' job assignments and criminal status, but also if they have a mindshield implant. Use this to your advantage in a revolution to definitively tell who is on your side!
+As a Security Officer, examining someone while wearing sechuds or HUDsunglasses will let you set their arrest level.
 As the Forensic Technician, you can dust almost any surface for prints. People leave fingerprints on everywhere they touch unless they are wearing gloves. However, as gloves leave fibres specific to their type, such as black or nitrile, it is possible to guess what general department they come from.
```
Comment (Doxxmedearly) [inline config/example/tips.txt:54]: Might want to check for any references to forensic technician and detective and replace them with Investigator

File: config/example/tips.txt [hunk 6]
```diff
@@ -12,78 +12,77 @@ You can grab someone by clicking on them with the grab intent, then upgrade the
...
+As a Cyborg, you are almost impervious to fires and heat. If you are rogue, you can release phoron fires everywhere and walk through them without a care in the world, if you can handle the bwoink afterwards.
+As a Cyborg, you are extremely vulnerable to EMPs as EMPs both stun you and damage you. Ion rifles or a traitor with an EMP kit can kill you in seconds.
+As the Chief Engineer, you can rename areas or create entirely new ones using your ship blueprints.
 As the Research Director, you can spy on and even forge PDA communications with the message monitor console! The key is in your office.
 As an Engineer, the supermatter crystal is an extremely dangerous piece of equipment: touching it will disintegrate you.
 As an Engineer, you can electrify grilles by placing wire "nodes" beneath them: the big seemingly unconnected bulges from a half completed wiring job.
 As an Engineer, you can lock emitters using your ID card to prevent others from disabling them.
 As an Atmospheric Technician, you can't unwrench a pipe if the pressure within is too high.
```
Comment (Doxxmedearly) [inline config/example/tips.txt:43]:
```suggestion
As an Atmospheric Technician, you can't unwrench a pipe if the pressure within is too high, unless you use a pipewrench.
```

File: config/example/tips.txt (hunk 1 above)
Comment (KingOfThePing) [reply config/example/tips.txt:68]: There were some on the Aurora. I wasn't sure. But I will fix.

File: config/example/tips.txt (hunk 5 above)
Comment (KingOfThePing) [reply config/example/tips.txt:54]: This is the only mention of FT. Detective was suprisingly not mentioned at all

File: config/example/tips.txt [hunk 7]
```diff
@@ -12,78 +12,76 @@ You can grab someone by clicking on them with the grab intent, then upgrade the
...
 As the Research Director, you can lock down cyborgs instead of blowing them up. Then you can have their laws reset or if that doesn't work, safely dismantled.
 As a Xenobiologist, you can maximize the number of uses you get out of a slime by feeding it slime steroid, created from purple slimes, while alive. You can then apply extract enhancer, created from cerulean slimes, on each extract.
-As a Scientist, researchable machine parts can seriously improve the efficiency and speed of machines around the station. In some cases, it can even unlock new functions.
-As a Scientist, the teleporter in Telescience can be set-up to teleport across the whole station! All you need to do is crack the formula.
+As a Scientist, researchable machine parts can seriously improve the efficiency and speed of machines around the ship. In some cases, it can even unlock new functions.
+As a Scientist, the teleporter in Telescience can be set-up to teleport across the whole sship! All you need to do is crack the formula, but be careful to not teleport into space.
```
Comment (JohnWildkins) [inline config/example/tips.txt:28]:
```suggestion
As a Scientist, the teleporter in Telescience can be set-up to teleport across the whole ship! All you need to do is crack the formula, but be careful to not teleport into space.
```

File: config/example/tips.txt [hunk 8]
```diff
@@ -12,78 +12,76 @@ You can grab someone by clicking on them with the grab intent, then upgrade the
...
+As the Head of Security, don't let the power go to your head. You may have high access, great equipment, and a miniature army at your side, but being a terrible person without a good reason is grounds for a bwoink.
 As the Warden, your duty is to be the watchdog of the brig and handler of prisoners when little is happening, and to hand out equipment and weapons to the security officers when a crisis strikes.
 As the Warden, keep a close eye on the armory at all times, as it is a favored strike point of mercenaries and cocky traitors.
 As the Warden, if a prisoner's crimes are heinous enough you can put them in permabrig. Make sure to check on them once in a while!
-As the Warden, you can implant criminals you suspect might re-offend with devices that will track their location and allow you to remotely inject them with disabling chemicals. 
+As the Warden, you can suggest to implant criminals you suspect might re-offend with devices that will track their location and allow you to remotely inject them with disabling chemicals.
```
Comment (JohnWildkins) [inline config/example/tips.txt:49]:
```suggestion
As the Warden, you can suggest implanting criminals you suspect might re-offend with devices that will track their location and allow you to remotely inject them with disabling chemicals.
```

**Review by JohnWildkins — APPROVED** [review]

**Pull request merged**